src/3rdparty/squirrel/sqstdlib/sqstdaux.cpp

```diff
@@ -38,7 +39,7 @@ void sqstd_printcallstack(HSQUIRRELVM v)
 					src = si.source;
 				}
 			}
-			pf(v,"*FUNCTION [%s()] %s line [" OTTD_PRINTF64 "]\n",fn,src,si.line);
+			pf(v,fmt::format("*FUNCTION [{}()] {} line [{}]\n",fn,src,si.line));
```

Comment:
As this diverges from the old Squirrel more, do we need to keep the no-spaces-after-commas coding-style?

Reply:
I tried to make as few changes as possible to make reviews easier.
It might be useful to "autoformat" all the squirrel code once with the updated formatting rules, to see whether that really helps. Then we might also consider formatting all the code with it (semi-automatically), i.e. run it through format in a CI step so it can complain about issues.